### ISSUE TITLE
Medley medley-full-wsl-x86_64-231217-3727653e_231112-42477318 causes error on Windopws 11

### ISSUE THREAD

**Issue by skaisler1** [Interlisp/medley#1485]
Once installed (?), trying to run medley.bat generates errors regarding scripts.

I downloaded the 231217 releases to my /home/skais/il directory.
I clicked on it to get WInzip, then told it to untar it.
I then extracted it which put files in ../il/medley as expected.
Files updated in ../il/medley show dates of 12/30 based on today's installation.
medley.bat is located in ../il.
I run it and get the following:
![image](https://github.com/Interlisp/medley/assets/39410155/adf4723e-4d5a-48f3-b21a-4e0efb8251dc)

I cannot examine medley/scripts/medley/medley.sh. Seems to have a lock on it.

I am runnning windows 11.

Steve K.

**Comment by fghalasz** [comment]
Steve,

The issues you are seeing are caused by EOL conversions (Linux->Windows) being done by WinZip.  It is possible to turn off these conversions in WinZip setting.  But you will then run into issues with WinZip not promulgating symbolic links for non-admin users.

Instead of any of the release tars, you should be using the Windows/Cygwin installer exe to update to the newest release.  Just download and run the cywin install exe for the release (e.g., medley-full-cygwin-x86_64-240114-e2764608_231112-42477318.exe) as you did for the initial install.  Use the same install directory and when it asks if you are sure you want to reuse an existing directory, answer in the affirmative.  This will install the new release over the old one.  (You could also specify a new install directory and just get a second independent copy of Medley on your machine.)

Using the cygwin install exe will avoid the myriad of issues that arise when trying to use a Windows app (e.g., WinZip) to install what are essentially Linux files.

-- Frank

**Comment by skaisler1** [comment]
Frank:
 
I have onlyused the Cygwin installer to overwrite the previous  contents of the directory.
So, here's the command:
 
 
So, medley.bat does not call run-medley, but I assume lde foes
 
lde is called from medley.sh which I cannt openas it seems to be locked by the system.
And, here's what I get:
 
 
 
Where does one provide parameters to run-medley since it is not in the call chain?
I am running on the latest version of Windows 11on a Dell Inspiron 15 3000 laptop
Notice there is no Medley icon.
 
If I went in and edited run-medley it wuould change the size of the window for Interlisp, but no icon.
 
Steve K.
